cmd/xmidt-agent/main.go

```diff
@@ -224,7 +226,7 @@ func onStart(cred *credentials.Credentials, ws *websocket.Websocket, logger *zap
 	}
 }
 
-func onStop(ws *websocket.Websocket, shutdowner fx.Shutdowner, eventCancelList []event.CancelFunc, pubsubCancelList []pubsub.CancelFunc, logger *zap.Logger) func(context.Context) error {
+func onStop(ws *websocket.Websocket, shutdowner fx.Shutdowner, eventCancelList []event.CancelFunc, pubsubCancelList []pubsub.CancelFunc, cancels []func(), logger *zap.Logger) func(context.Context) error {
```

Comment:
in a separate pr, I want to simplify this to `onStop(ws *websocket.Websocket, shutdowner fx.Shutdowner, cancels []func(), logger *zap.Logger) func(context.Context)`, replacing `eventCancelList` and `pubsubCancelList` with `cancels`.

https://github.com/xmidt-org/xmidt-agent/issues/109